pre_purge_check.csh

```diff
@@ -0,0 +1,46 @@
+#!/bin/tcsh
+
+# Display the destination files from repack_st_arch.csh.
+# This can't be rolled into repack_st_arch.csh
+# because that submits globus transfer jobs
+# that take an unknown amount of time to finish.
+
+# On the data-access node.
+
+if ($#argv == 0) then
+   set y = 2017
+   set m = 1
+else if ($#argv == 2) then
+   set y = $1
+   set m = $2
+else
+   echo "Usage: after running case.st_archive and repack_st_archive,"
+   echo "       on casper:"
+   echo "       % pre_purge.csh yyyy m"
+   exit
+endif
```

Comment:
I would prefer that the script get called exactly one way ... calling it with no arguments seems like we could be getting the wrong year without really knowing it.